internal/test/framework/windowsvm.go

```diff
@@ -162,10 +168,39 @@ func (w *windowsVM) Run(cmd string, psCmd bool) (string, string, error) {
 	return stdout.String(), stderr.String(), nil
 }
 
+func (w *windowsVM) RunOverSSH(cmd string, psCmd bool) (string, error) {
```

Comment:
Please add go docs for this, also mention in the comments why is this method needed in first place despite having winrm

Reply:
I thought the convention was to add the go doc in the interface and not in the implementation. I will update the comment there saying why it is needed.

Reply:
Yeah, anywhere is fine. As long as we have info on why and when the function is needed.